internal/cmd/beta/network/update/update_test.go

```diff
@@ -80,9 +80,6 @@ func fixturePayload(mods ...func(payload *iaas.PartialUpdateNetworkPayload)) iaa
 			Ipv4: &iaas.UpdateNetworkIPv4{
 				Nameservers: utils.Ptr([]string{"1.1.1.0", "1.1.2.0"}),
 			},
-			Ipv6: &iaas.V1UpdateNetworkIPv6{
```

Comment:
And then this could be re-added, as currently it's excluding it from the payload even though it's specified in the input model